### DISCUSSION TITLE
coc-settings.json symlink not being read properly

### DISCUSSION THREAD

**Discussion by FA5I** [neoclide/coc.nvim#4376] [Q&A] [answered]
Hi,

I keep my config in a folder that I checkin to git to make it easy to access from other computers.

I have created a symlink to the `coc-settings.json` file:

However, when running `CocConfig` in nvim, it seems that the file is empty. Any idea why that is? I don't think it's a bug, just something I'm possibly messing up.

Any help would be much appreciated!

<img width="1751" alt="Screenshot 2022-11-20 at 15 43 36" src="https://user-images.githubusercontent.com/56031768/202911498-22c8e87a-f990-49b6-a46f-2ff6304e5de9.png">

**Comment by chemzqm** [comment, marked as answer]
Can't reproduce, could be permission issue.
